provisioning/plugins/corim-psa-decoder/ccaplatformconfigid.go

```diff
@@ -33,19 +33,26 @@ func (o *CCAPlatformConfigID) FromMeasurement(m comid.Measurement) error {
 	return nil
 }
 
-func (o CCAPlatformConfigID) MakeRefAttrs(c PSAClassAttributes) (*structpb.Struct, error) {
+func (o CCAPlatformConfigID) GetRefValType() string {
+	return "platform-config"
+}
+
+// MakeRefAttrs is an interface function to populate reference attributes.
+// For PlatformConfig scheme argument is not strictly required, but is required for other
+// usage of the same interface
+func (o CCAPlatformConfigID) MakeRefAttrs(c ClassAttributes, scheme string) (*structpb.Struct, error) {
```

Comment:
I thought we agreed to rename `scheme` to `unused` and hardcode the required value in L45-47

Reply:
As far as I recollect we agreed to retain the usage but with a clarification that it is strictly not required.
This is more clear than using the hard coded one...